utilities/prepare.sh

```diff
@@ -7,7 +7,7 @@ echo It will also attempt to build the benchmarks
 echo It will output OK at the end if builds succeed
 echo
 
-IOR_HASH=io500-sc19
+IOR_HASH=a69a5916cf0a057
```

Comment:
Please revert this. Let me bump the hash at the IOR repository to a69a5916cf0a057 As we figured there is an issue that affects everyone.

Reply:
Done